make/framework.mk

```diff
@@ -128,15 +126,6 @@ test_warnings: $(FRAMEWORK_HEADER)
 		-fno-objc-arc $@
 	@rm $(FRAMEWORK_HEADER:%.h=%.o)
 
-$(MODULE_MAP):
```

Comment:
The module map looks useful for Swift interop purpose. If we remove them here, does that mean we need to import it somewhere again when necessary?

Reply:
I'm running some experiments to get to a somewhat definitive answer to this question. I'll get back to you by the end of the week.

Reply:
@litstrong: I ran a number of experiments and I couldn't figure out a way to define a static library which consists of Objective-C classes only such that it can be imported as a module into another module. The only way that I could get the library's headers to become visible to Swift classes in the importing module was to add the library's headers to the importing module's Objective-C Bridging Header.

I compared the build outputs produced by a static library which consists of Objective-C classes only against the build outputs produced by a static library which contains of one or more Swift classes. I'll summarise what I saw for each.

In the case of a static library which consists of Objective-C classes only, the build outputs were as follows:

```none
+-- libMyLibrary.a (file)
\-- include (directory)
  \-- MyLibrary (directory of headers)
```

In the case of a static library which contains one or more Swift classes, the build outputs were as follows:

```none
+-- libMyLibrary.a (file)
\-- MyLibrary.swiftmodule (directory)
```

The `swiftmodule` directory seems to me to be the key to allowing a library to be imported as a module in the importing module's Swift code. If a library does not produce a `swiftmodule` directory, the way to use the library's Objective-C classes in the importing module's Swift code is to add the library's headers to the importing module's Objective-C Bridging Header.